test/python/test_lapse_rate.py

```diff
@@ -146,7 +146,7 @@ def execute(self, port, data_in, req_in):
 
 
 def isequal(a, b, epsilon):
-    return np.fabs(a - b) < epsilon
+    return ((a - b)**2)**(0.5) < epsilon
```

Comment:
You could use numpy's isclose or allclose for a more robust implementation. Note: TECA has C++ implementation in teca_coordinate_util::equal for use in C++ code.